src/Modules/CalcDefence.lua

```diff
@@ -257,40 +257,9 @@ function calcs.reducePoolsByDamage(poolTable, damageTable, actor)
 	}
 end
 
--- Performs all ingame and related defensive calculations
-function calcs.defence(env, actor)
+function calcs.resistances(actor)
 	local modDB = actor.modDB
-	local enemyDB = actor.enemy.modDB
 	local output = actor.output
-	local breakdown = actor.breakdown
-
-	local condList = modDB.conditions
-
-	-- Action Speed
-	output.ActionSpeedMod = calcs.actionSpeedMod(actor)
-	
-	-- Armour defence types for conditionals
-	for _, slot in pairs({"Helmet","Gloves","Boots","Body Armour","Weapon 2","Weapon 3"}) do
-		local armourData = actor.itemList[slot] and actor.itemList[slot].armourData
-		if armourData then
-			wardBase = armourData.Ward or 0
-			if wardBase > 0 then
-				output["WardOn"..slot] = wardBase
-			end
-			energyShieldBase = armourData.EnergyShield or 0
-			if energyShieldBase > 0 then
-				output["EnergyShieldOn"..slot] = energyShieldBase
-			end
-			armourBase = armourData.Armour or 0
-			if armourBase > 0 then
-				output["ArmourOn"..slot] = armourBase
-			end
-			evasionBase = armourData.Evasion or 0
-			if evasionBase > 0 then
-				output["EvasionOn"..slot] = evasionBase
-			end
-		end
-	end
```

Comment:
Moving thie block breaks support for the Armour Mastery that grants max res when you have armour on each piece of gear

Reply:
In that case, the minimal change is probably to leave a call to the extracted `calc.resistances` function where the code was originally in `calc.defence` and just have minions calculate resists twice. It's not elegant but would get the job done. Do you think that's ok or should I hold off on this until there's a larger refactor (which I can take a stab at)